src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -1578,6 +1601,24 @@ private boolean isUserValid(final RegisteredUser userToValidate) {
         return isValid;
     }
 
+    /**
+     * This function checks that the name provided is valid.
+     *
+     * @param name
+     *            - the name to validate.
+     * @return true if the name is valid, false otherwise.
+     */
+    public boolean isUserNameValid(final String name){
```

Comment:
This could be `final` to be more explicit but having looked at the methods around it, they aren't either...

Reply:
Added.